unlock-app/src/components/creator/CreatorLockForm.js

```diff
@@ -44,6 +45,11 @@ class CreatorLockForm extends React.Component {
     if (this.props.hideAction) this.props.hideAction()
   }
 
+  handleCancel(e) {
+    e.stopPropagation() // This prevents submit from also being called
```

Comment:
I think we actually want `preventDefault` because it's less restrictive. [See this](https://stackoverflow.com/questions/5963669/whats-the-difference-between-event-stoppropagation-and-event-preventdefault).